app/controllers/calagator/application_controller.rb

```diff
@@ -92,5 +97,11 @@ def recaptcha_verified?(model)
 
       true
     end
+
+    def community_phrase_verified?(model)
+      return verify_recaptcha(model: model) if recaptcha_enabled?
```

Comment:
Did you mean to call the verify_recaptcha method inside here?

Reply:
I did not. I'll fix it now.